docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
```

Comment:
I know Julien used version 3.3 in his slides. I looked into the different versions and I don't think there is much of a difference between 3.0 and 3.3 (for our purposes atleast), but I just wanted to let you know in case you wanted to be consistent with his.